src/EnergyPlus/SimulationManager.cc

```diff
@@ -1150,6 +1150,23 @@ namespace SimulationManager {
             DoWeathSim = true;
         }
 
+        CurrentModuleObject = "PerformancePrecisionTradeoffs";
+        int PerfPrecTradeoffs = inputProcessor->getNumObjectsFound(CurrentModuleObject);
+        if (PerfPrecTradeoffs > 0) {
+            inputProcessor->getObjectItem(CurrentModuleObject,
+                                          1,
+                                          Alphas,
+                                          NumAlpha,
+                                          Number,
+                                          NumNumber,
+                                          IOStat,
+                                          lNumericFieldBlanks,
+                                          lAlphaFieldBlanks,
+                                          cAlphaFieldNames,
+                                          cNumericFieldNames);
+            if (Alphas(1) == "YES") DoCoilDirectSolutions = true;
+        }
```

Comment:
Should grab the JSON directly...

Reply:
@Myoldmopar I followed the pattern. Can you give a reference example for this? Thanks.

Reply:
```
auto const instances = inputProcessor->epJSON.find(PerformancePrecisionTradeoffs);
if (instances != inputProcessor->epJSON.end()) {
    auto &instancesValue = instances.value();
    for (auto instance = instancesValue.begin(); instance != instancesValue.end(); ++instance) {
        auto const &fields = instance.value();
        auto const &thisObjectName = UtilityRoutines::MakeUPPERCase(instance.key());
        inputProcessor->markObjectAsUsed(thisObjectType, thisObjectName);
        if (fields.find("do_direct_coil_solution") != fields.end()) {
            if (UtilityRoutines::MakeUPPERCase(fields.at("do_direct_coil_solution")) == "YES") {
                DoCoilDirectSolutions = true;
            }
        }
    }
}
```